tripal_chado/tests/src/Kernel/Plugin/TripalBackendPublish/ChadoPublishTest.php

```diff
@@ -334,6 +335,26 @@ public function testTripalPublishServiceSingleJob() {
       $this->assertEquals($expected, $this->countFieldTable($field_table),
         'The drupal field table does not contain the expected number of publications for the analysis');
     }
+    // Tests unpublishing.
```

Comment:
We will want to check the Drupal and chado databases as well to confirm the correct records are being deleted (those in Drupal) and also others are left alone (those in chado).

For this first test where there are no orphans, you can just to a count on the tables to confirm the drupal and chado table record counts have not changed.